docs/how-tos/rotating-secrets.md

```diff
@@ -0,0 +1,102 @@
+  * In our case, this was leaked in a user comment within a GitHub issue.
+
+
+## Fix the leak
+
+The most important thing to do -- particularly if the secret is valid -- is to remove the exposed secret from wherever it was found.
+
+In this example, the secret was leaked in a GitHub issue comment. The comment can be edited to redact the reference, but it will still be retained in the comment's revision history. An [admin for the repo](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization) can remove the revision which introduced the leak to completely remove any public reference to it.
+
+## Rotate the secret (if valid)
+
+Now that we've stopped the bleeding, it's time to rotate the secret if it was determined to be valid and in use. 
+
+The steps here are to:
+1. create a new secret,
+2. replace the old secret where it's used with the new secret, and
+3. invalidate the old secret
+
+:::warning
+Order matters! Don't invalidate the old key unless/until a new key has been created, the old key's value has been replaced everywhere it's used and any affected parties (such as other engineers) have been notified.
+
+Depending on the urgency and deployment process, this overlap period where both keys are valid could be on the order of seconds, hours or even days. Discuss this with the team if you're uncertain.
+:::
+
+For our example, [Stripe provides some helpful docs](https://stripe.com/docs/keys#rolling-keys) on how to create a new secret and invalidate the old secret. The hard part here is replacing the old secret where it is used with the new secret, as we store secrets in a number of different places for different purposes.
+
+### Places to check where a secret might be used
+* Application config environment variables
+  * These secrets are managed by SRE, so you will need to [securely send them the new secret](https://mozilla-hub.atlassian.net/wiki/spaces/SECENGOPS/pages/378273870/How+to+share+a+secret+with+coworkers), so they can update it and redeploy the affected application(s).
+  * If you're unsure what environment variable(s) represent this secret in what app(s), you can send them both the old secret and the new secret, so they can do a find-and-replace.
+* [CircleCI environment variables](https://circleci.com/docs/env-vars/#private-keys-and-secrets)
+* GitHub [Actions](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions), [Codespaces](https://docs.github.com/en/codespaces/managing-your-codespaces/managing-secrets-for-your-codespaces) and/or [Dependabot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot) environment variables
+
+:::caution
+This is not an exhaustive list. Check with other engineers and/or SRE if you are unsure where all a secret may be kept.
```

Comment:
Nit: reword. Grammar appears a bit off here:

> if you are unsure where all a secret may be kept.

Reply:
Reworded to "if you are unsure of where all of the locations where a secret may be kept".